apps/src/templates/instructions/InlineAudio.jsx

```diff
@@ -175,9 +176,18 @@ const InlineAudio = React.createClass({
         <div
           className="inline-audio"
           style={this.props.style && this.props.style.wrapper}
+          onMouseOver={function (){
+            $("#volume").css("background-color", color.cyan);
+            $(".playPause").css("background-color", color.cyan);
+          }}
+          onMouseOut={function (){
+            $("#volume").css("background-color", color.lightest_purple);
+            $(".playPause").css("background-color", color.lightest_purple);
+          }}
```

Comment:
I think it would be preferable to set a hover state and that in turn enabled a hover style on the inner components here, rather than mixing jQuery-stateful changes with React-stateless components.  @Hamms opinion here?